.ci/docker/requirements-ci.txt

```diff
@@ -263,7 +263,8 @@ rockset==1.0.3
 #Pinned versions: 1.0.3
 #test that import:
 
-ghstack==0.7.1
+ghstack==0.7.1 ; python_version < "3.12"
+ghstack==0.8.0 ; python_version >= "3.12"
```

Comment:
Why can't we use latest ghstack for all?

Reply:
This comes from original PR:
https://github.com/pytorch/pytorch/pull/113987/commits/e1928fd9737800842c2d3ae053f9f11d80ff769b